product-scenarios/2-Collaborative-API-Development/README.md

```diff
@@ -0,0 +1,8 @@
+# 02. Collaborative API Development
+
+## Sub Scenarios
+
+### [2.1-Manage-publisher-portal-operations-for-users](https://github.com/wso2/product-apim/tree/product-scenarios/product-scenarios/2-Collaborative-API-Development/2.1-Managing-publisher-portal-operations-for-users)
```

Comment:
I think we can remove hyphens between words in this level.

Reply:
Please also update https://github.com/wso2/product-apim/blob/product-scenarios/product-scenarios/README.md